### ISSUE TITLE
Use other types than `Buffer` for `Bytes` field

### ISSUE THREAD

**Issue by PatrikElfstrom** [prisma/prisma#12987]
## Problem

Prisma is using `Buffer` for the `Bytes` field and because of that I need to convert the `Buffer` to `Uint8Array` before returning the data to be able to use it in the browser.
And since I'm using typescript I want to be able to reuse the types without having to type cast the `Uint8Array` back to `Buffer`.

```TypeScript
const files = Prisma.file.findMany();
return files.map(file => {
  file.blob = new Uint8Array(file.blob) as Buffer;
  return file;
}
```

This feel like an unnecessary hack.

Initial issue: blitz-js/superjson#181

## Suggested solution

Allow to use another type like `Uint8Array` instead of `Buffer` for the `Bytes` field. 

## Alternatives

Allow any other of the types supported by the ECMAScript Specification.
https://tc39.es/ecma262/multipage/indexed-collections.html#table-49

**Comment by JonathanWilbur** [comment]
In my opinion, it's just good programming to define functions and interfaces to use `Uint8Array` instead of `Buffer` unless you specifically need something that is in `Buffer`, but not `Uint8Array`. (`Buffer` inherits from `Uint8Array` in case that wasn't clear.) I think all Prisma does with buffers supplied to queries is write them to the wire, in which case, `Uint8Array` should suffice.

The only thing I would warn surrounding this is that there is a big (and dangerous) difference between `Uint8Array.buffer` and `Buffer.buffer`!

**Comment by andyjy** [comment]
Now implemented in Prisma v6 https://github.com/prisma/prisma/pull/25719 
Good to close I think(?) @aqrln 

(Discovered while searching for clues for a related issue)

**Comment by aqrln** [comment]
Yes, good catch, thank you. We changed `Buffer` to `Uint8Array` in Prisma 6.
